docs/ora_team_submissions.rst

```diff
@@ -0,0 +1,91 @@
+ORA2 Team Submissions
+=====================
+
+.. contents::
+
+Workflows
+---------
+
+There’s an AssessmentWorkflow model that has a workflow app label, which means that the table name is prefixed with `workflow_` and not `assessment_` like everything else (I’m sure it’s to help resolve a name collision).  `See for reference: <https://github.com/edx/edx-ora2/blob/001a6804e605426fdc43d4b5d83a38051c756c8d/openassessment/workflow/models.py#L108>`_.  In fact, each of the AssessmentWorkflow, AssessmentWorkflowStep, and AssessmentWorkflowCancellation models live under the workflow app label.
+
+- The ``SubmissionMixin.create_submission()`` method calls the ``workflow_api.create_workflow()`` method  (it stops at ``WorkflowMixin.create_workflow()`` along the way).  This is the entry point to workflow management in ORA.
+
+- ``create_workflow()`` calls ``AssessmentWorkflow.start_workflow()`` 
+
+ - …which calls the ``on_init()`` method of each specific type of step (e.g. self, peer, staff).
+
+ - The ``on_init()`` method of e.g. StaffWorkflow is what creates a StaffWorkflow record associated with a given course, item, and submission (and eventually, a scorer and an assessment).
+
+
+Staff Workflows
+---------------
+
+These should be the only workflow type that we care about for team submissions.
+
+`Python API for managing StaffWorkflow data: <https://github.com/edx/edx-ora2/blob/master/openassessment/assessment/api/staff.py>`_:
+
+- **on_init(submission_uuid)**
+
+ - Looks up submission record from Submissions API.  Then creates a StaffWorkflow associated with it.
```

Comment:
Formatting for this shows as a block quote w/ bulleted list instead of correctly indenting.